### PR TITLE
Requires PHP 7.1+

### PR DESCRIPTION
**Pull request by olvlvl**
@red-led Wdyt?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,11 @@
 		"source": "https://github.com/ICanBoogie/Storage"
 	},
 	"require": {
-		"php": ">=5.5.0"
+		"php": ">=7.1"
+	},
+	"require-dev": {
+		"ext-json": "*",
+		"ext-apcu": "*"
```
Comment (red-led) [inline composer.json]: Will installation fail if i don't have `ext-apcu`?

Comment (olvlvl) [reply composer.json]: The requirement only applies to local development, to be able to run the test suite. The extension is not required to use the library. `require-dev` is root only, it only applies to the root package context.

https://getcomposer.org/doc/04-schema.md#require-dev

Comment (red-led) [reply composer.json]: Ah... yep, makes sense.

File: Makefile [hunk 2]
```diff
@@ -40,6 +40,15 @@ test-coveralls: test-dependencies
...
 
+test-container:
+	@docker-compose \
+		-p icanboogie-storage-test \
+		run --rm app bash
+
+	@docker-compose \
+		-p icanboogie-storage-test \
+		down -v
+
```
Comment (red-led) [inline Makefile]: Do i need docker to run tests locally?

Comment (olvlvl) [reply Makefile]: Provided you have the required extensions, you can run the tests on your machine. If you don't, the Docker image provides a reproducible stable environment. If you're not familiar with Docker, I recommend you give it a try, it's quite simple.

**Comment by red-led** [comment]
Also, i know you are trying to have minimum deps, but why do not put PhpUnit into `require-dev`? It will simplify your makefile script.

**Comment by olvlvl** [comment]
Just because it's faster to download the PHAR than resolve and install dependencies with composer.

**Pull request merged**